Sources/CodeEditTextView/STTextViewController.swift

```diff
@@ -342,3 +342,30 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
         highlighter = nil
     }
 }
+
+class CEScrollView: NSScrollView {
```

Comment:
As per your comment, you could put this in the root of the package. We don't have enough view types to require a folder yet so it should live there.